grpc/service_config/service_config.proto

```diff
@@ -202,13 +202,18 @@ message WeightedRoundRobinLbConfig {
   // the endpoints. By default, per-request utilization reporting is used.
   google.protobuf.BoolValue enable_oob_load_report = 1;
 
+  // Load reporting interval to request from the server.  Note that the
+  // server may not provide reports as frequently as the client requests.
+  // Used only when enable_oob_load_report is true.
```

Comment:
"Defaults to 10 seconds."?

Reply:
Yup, already did that -- you must not have been looking at the latest commit.